source/extensions/filters/common/rbac/utility.h

```diff
@@ -12,6 +12,8 @@ namespace Filters {
 namespace Common {
 namespace RBAC {
 
+static const std::string shadow_policy_id_field = "shadow_effective_policyID";
```

Comment:
I know this were here before, but this will potentially introduce static initialization order fiasco, can you make this in a ConstSingleton, or as absl::string_view (as it has constexpr constructor)?